### PR TITLE
project ready for marking

### PR DESCRIPTION
**Pull request by qkhn95**
1### Work Done
2 - copied assets and index from main instructor folder
3 - pasted into homework folder

4### TO DO
5 - start to find code which is broke
6 - update the code using semantic html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <!----------title changed--------->
+    <title>Homepage -HORISEON Social Solution Services</title>
+  </head>
+  <!----Title End---->
+
+  <!----Body start---->
+  <body>
+    <!-----Header---->
+    <div class="header">
```
Comment (chelseanicholls) [inline index.html:15]: There is a semantic tag that you can use here. Take a look at [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more information about the header tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            src="./assets/images/cost-management.png"
+            alt="Cog with dollar logo"
+          />
+          <p>
+            As the search ranking for your business increases, your advertising
+            costs decrease, and you no longer need to advertise your page.
+          </p>
+        </div>
+      </section>
+    </main>
```
Comment (chelseanicholls) [inline index.html:121]: The `main` tag should be closed before the benefits section as the benefits are not part of the main content.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          <p>
+            Social media continues to have a sizable influence on buying habits.
+            Social media marketing helps you determine which platforms are
+            suited to your brand, using analytics to find the right markets and
+            increase your lead generation.
+          </p>
+        </section>
+      </div>
+
+      <section class="benefits">
```
Comment (chelseanicholls) [inline index.html:89]: This could be an `aside` tag instead of a section.

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,134 @@
+* {
```
Comment (chelseanicholls) [inline assets/css/style.css:1]: Remember to add comments to your CSS file

File: index.html (hunk 3 above)
Comment (qkhn95) [reply index.html:89]: I have now changed the section tag to aside

File: index.html (hunk 2 above)
Comment (qkhn95) [reply index.html:121]: I have made the change here.

**Comment by qkhn95** [comment]
Sorry for the late reply. I have made some of the changes you have requested. I am happy with the grade received also so will not be asking for regrading. Thank you for suggesting the changes.

File: index.html (hunk 1 above)
Comment (qkhn95) [reply index.html:15]: Hi Chelsea, sorry for the late reply. I have made some of the changes you have requested. Right now I am happy with my grade so will not be asking for a re-grade. Thank you for the suggestions as they are really helpful.

**Pull request merged**